src/index.html

```diff
@@ -5,10 +5,69 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    
     <title>Moyo header</title>
+    
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+ 
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      
+      <a href="#home" class="logo"> 
+        <img src="images/logo.png" alt="logo" class="logo"> 
```

Comment:
```suggestion
      <a href="#home"> 
        <img src="images/logo.png" alt="logo" class="logo"> 
```
Для чого тут два однакових класа?